src/app/component/CartItem/CartItem.container.js

```diff
@@ -46,6 +46,9 @@ export const mapDispatchToProps = (dispatch) => ({
     removeProduct: (options) => CartDispatcher.then(
         ({ default: dispatcher }) => dispatcher.removeProductFromCart(dispatch, options)
     ),
+    updateCrosssellProducts: (items) => CartDispatcher.then(
```

Comment:
So many 's' letters :D
It'd be better to `updateCrossSellProducts`

Reply:
Done